src/screens/CreateEventPage.js

```diff
@@ -28,20 +27,20 @@ class CreateEventPage extends Component {
 
       //Name
       if (!fields["eventNameText"]) {
-         this.state.formIsValid = false;
+         this.state.formIsValid.setState(false);
          errors["eventNameText"] = "Please enter an event name";
       }
 
       if (typeof fields["eventNameText"] !== "undefined") {
-         if (!fields["eventNameText"].match(/^[\S\s]{3,25}$/)) {
-            this.state.formIsValid = false;
+         if (!fields["eventNameText"].match(/^[\S\s]{3,15}$/)) {
+            this.state.formIsValid.setState(false);
```

Comment:
All of these `this.state.formIsValid.setState()` statements are not valid syntax and break the site. If you want to set the `formIsValid` state then you need `this.setState({formIsValid: false})`

Reply:
thanks, forgot to change them back.